Example/App/App.js

```diff
@@ -17,7 +17,9 @@ const colors = [
   '#4CA0F7',
   '#123456',
   '#F87217',
+  '#9090ee'
 ];
+
 export default class App extends Component {
```

Comment:
This is still a class and it needs to be a function component

Reply:
Done